src/parser/jobs/rpr/modules/HarvestMoon.tsx

```diff
@@ -0,0 +1,149 @@
+	private currentSoulSow?: SoulSowData
+
+	override initialise() {
+		super.initialise()
+
+		const playerFilter = filter<Event>().source(this.parser.actor.id)
+
+		this.addEventHook(playerFilter.type('statusApply').status(this.data.statuses.SOULSOW.id), this.onHarvestMoonStatusApply)
+		this.addEventHook(playerFilter.type('statusRemove').status(this.data.statuses.SOULSOW.id), this.onHarvestMoonStatusRemove)
+		this.addEventHook(playerFilter.type('action').action(this.data.actions.HARVEST_MOON.id), this.onHarvestMoonUsage)
+
+		this.addEventHook('complete', this.onComplete)
+	}
+
+	private onHarvestMoonStatusApply(event: Events['statusApply']) {
+		if (this.currentSoulSow == null) {
+			this.currentSoulSow = {
+				used: false,
+				timestamp: event.timestamp,
+			}
+		}
+	}
+
+	private onHarvestMoonStatusRemove() {
+		this.onHarvestMoonUsed(false)
+	}
+
+	private onHarvestMoonUsage() {
+		this.onHarvestMoonUsed(true)
+	}
+
+	private onHarvestMoonUsed(used: boolean) {
+		if (this.currentSoulSow == null) {
+			if (used) {
+				this.brokenLog.trigger(this, 'rpr.harvest-moon.trigger.use-before-soulsow', 'Harvest moon were used prior to using Soul Sow skill')
+			}
+
+			return
+		}
+
+		this.currentSoulSow.used = used
+		this.soulSowData.push(this.currentSoulSow)
+		this.currentSoulSow = undefined
+	}
+
+	private onComplete() {
+		const downtimeWindows = this.downtime.getDowntimeWindows()
+
+		const requirements: Requirement[] = []
+
+		const prePullUsage = this.soulSowData.find(d => d.timestamp <= this.parser.pull.timestamp)
+
+		requirements.push(this.createSoulSowChecklistRequirement(
+			<Trans id="rpr.harvest-moon.checklist.requirement.pre-pull.name">Usage on pre-pull</Trans>,
+			prePullUsage)
+		)
+
+		if (downtimeWindows.length > 0) {
+			downtimeWindows.forEach(dw => {
+				const duration = dw.end - dw.start
+				const isValidWindow = duration > this.SOUL_SOW_CAST_TIME
+
+				// if the window was enough to cast soul sow check if we actually did
+				if (isValidWindow) {
+					const soulSowUsage = this.getSoulSowCastOnWindow(dw.start, dw.end)
+
+					const startTime = this.parser.formatEpochTimestamp(dw.start)
+					const endTime = this.parser.formatEpochTimestamp(dw.end)
+
+					requirements.push(this.createSoulSowChecklistRequirement(
+						<Trans id="rpr.harvest-moon.checklist.requirement.downtime.name">Usage on downtime
+							window {startTime} to {endTime}</Trans>,
+						soulSowUsage
+					))
+				}
+			})
+
+			this.checklist.add(new Rule({
+				name: <Trans id="rpr.harvest-moon.checklist.name">Use <ActionLink action="SOULSOW"/> during pre-pull and
+					downtime</Trans>,
+				description: <Trans id="rpr.harvest-moon.checklist.description">The <ActionLink action="SOULSOW"/> skill
+					have a very high potency but also high cast time during battle, consider casting it before the fight
+					starts and also when the fight is in a downtime.
+					Don't forget to use <ActionLink action="HARVEST_MOON"/> before a downtime.</Trans>,
```

Comment:
Soulsow itself doesn't have any potency, but rather unlocks Harvest Moon (which is a high potency attack).  Maybe change the wording and try to be more active with it, such as:

```suggestion
				description: <Trans id="rpr.harvest-moon.checklist.description">Using <ActionLink action="SOULSOW" /> before the pull or during fight downtime unlocks the high potency attack <ActionLink action="HARVEST_MOON" />.  Be sure to use each buff gained from <ActionLink action="SOULSOW" />, but only cast <ActionLink action="SOULSOW" /> when you otherwise cannot attack any enemies for at least 5 seconds.</Trans>,
```

Reply:
That sounds good!